drivers/ethernet/intel/eth_intel_igc.c

```diff
@@ -656,9 +656,9 @@ static sa_family_t eth_intel_igc_get_sa_family(uint8_t *rx_buf)
 
 	switch (ntohs(eth_hdr->type)) {
```

Comment:
we should have a non-POSIX version of `ntohs()` and others too.

Reply:
Yes, I will change these. I started to change the source all over the code base but there was just so much of it, so I decided to concentrate my efforts first to subsys/net, tests/net and include/zephyr/net directories. Some of the drivers contain already changes and I will continue this work.